src/containers/App.js

```diff
@@ -0,0 +1,5 @@
+import { connect } from 'react-redux';
+import App from '../components/App';
+
+
+export default connect()(App);
```

Comment:
please add an empty line as last line in every file (eslint)

Reply:
@ChristianSchmidt1981 
It has the empty line but it is not showing up.